html/admin/project/twigIncludes/assetBoard/board.twig

```diff
@@ -1,3 +1,9 @@
+{% if "PROJECTS:PROJECT_ASSETS:EDIT:ASSIGNMENT_STATUS"|instancePermissions and instanceid == USERDATA.instance["instances_id"] %}
+<button type="button" class="btn btn-default" title="Scan Assets" id="scanAssets" data-toggle="modal" data-target="#scanAssetModal">
```

Comment:
This would look better if it was in the tab bar above, like we have in the Assets list tab - eg one of the icons here: 
![image](https://github.com/adam-rms/adam-rms/assets/54247748/e7e0da41-b93c-47b3-a43f-f1a634bb77e6)

I can see why they're per tab, but it feels like a bit of a 'tack this on the top here' atm

Reply:
Moved this into the tab bar on the dispatch screen: 
<img width="642" alt="image" src="https://github.com/adam-rms/adam-rms/assets/2904440/77f5724b-9300-448f-9ef1-9adadde88511">